modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItLocksSystemViewTest.java

```diff
@@ -83,8 +83,16 @@ public void testMetadata() {
     public void testData() {
         Ignite node = CLUSTER.aliveNode();
 
-        sql("CREATE TABLE test (id INT PRIMARY KEY, val INT)");
-        sql("INSERT INTO test VALUES (0, 0), (2, 2)");
+        // Need 3 different tables here, because near sql flow: INSERT -> UPDATE -> DELETE leads to dead lock
```

Comment:
can we add todo with a ticket for a fix? in some time the issue was gone, but the comment will still be there and will lead to incorrect understanding.

Reply:
test was reworked.